### ISSUE TITLE
Is $ a valid module name?

### ISSUE THREAD

**Issue by filipenevola** [benjamn/reify#243]
Hi @benjamn, we got this issue on Meteor https://github.com/meteor/meteor/issues/11279

I believe it is because X6 is using `import $` and then reify is breaking in this case.

Am I right? Is this a valid module name?

**Comment by benjamn** [comment]
@filipenevola Yes, `$` is a valid identifier (variable name), so it should be importable/exportable.
